packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -199,7 +208,9 @@ export class FabricTestLedgerV1 implements ITestLedger {
     const fnTag = `${this.className}#getConnectionProfileOrg1()`;
     const cInfo = await this.getContainerInfo();
     const container = this.getContainer();
-    const ccpJsonPath = "/fabric-samples/first-network/connection-org1.json";
+    const ccpJsonPath = this.getFabricVersion().valueOf().includes("1.4.")
+      ? "/fabric-samples/first-network/connection-org1.json"
```

Comment:
Pull these magic strings out to separate constant variables where the variables of the names show what they hold and which fabric version they are used for (good variable/method names make the code basically self-documenting).

Reply:
Incorporated the changes. Please have a re-look.